docs/openapi.yaml

```diff
@@ -14,30 +14,59 @@ info:
 
     It is comprised of two parts; the Stacks Blockchain API and the Stacks Core API.
 
-    <a href="collection.json" download="collection.json">Download Postman collection</a>
+    <a href="https://hirosystems.github.io/stacks-blockchain-api/collection.json" download="collection.json">Download Postman collection</a>
```

Comment:
I assume you're setting the absolute URL because it's now hosted on the new docs site, right?
If so, you should also set the URL in the `download` attribute.

Reply:
The value on the download attribute is actually optional and all it does is specify what the filename will be when it's downloaded. Since we're not changing the filename, I'll just remove it and save the space.

Reply:
I see. I don't think you should remove it, though. IIRC when I was adding that link I tested on Safari and Chrome and without the `download` attribute, the JSON file was opened as a text file in the current tab rather than a file download.

Reply:
Right - I'm only removing the value on the attribute. The final line of HTML looks like this `<a href="https://hirosystems.github.io/stacks-blockchain-api/collection.json" download>Download Postman collection</a>`

Reply:
Ah, perfect 👍